src/backend/api/meals.js

```diff
@@ -1,12 +1,79 @@
 const express = require("express");
+const { request, response } = require("../app");
 const router = express.Router();
 const knex = require("../database");
 
+// maxPrice
 router.get("/", async (request, response) => {
+  let meals = knex("meal");
+  if ("maxPrice" in request.query) {
+    if (!"maxPrice") {
+      response.status(403).json("Please enter data");
```

Comment:
I would make this a status 400 and then respond with an object like `{ error: "Please enter data" }`.

Also, "Please enter data" is not a great error message as it is unclear what the issue is 😄